src/notebooklm/_android/proto_src/google/internal/labs/tailwind/orchestration/v1/orchestration_service.proto

```
@@ -45,6 +45,8 @@ service LabsTailwindOrchestrationService {
   rpc AddTentativeSources(AddTentativeSourcesRequest)
       returns (AddTentativeSourcesResponse);
   rpc AddSources(AddSourcesRequest) returns (AddSourcesResponse);
+  rpc ListExpertIntelligenceContent(ListExpertIntelligenceContentRequest)
```

Buf (1.72.0)

[error] cannot find `ListExpertIntelligenceContentRequest` in this scope

(COMPILE)